src/components/LocationCard/LocationCard.js

```diff
@@ -74,6 +77,8 @@ LocationCard.propTypes = {
     onRadiusChange: PropTypes.func,
     /** Location card delete callback */
     onDelete: PropTypes.func,
+    /** Name of the wrapping component */
+    As: PropTypes.string,
```

Comment:
I suggested a little bit change comment for transparency `/** HTML tag to be used to for the wrapping component */`